lib/rust/ensogl/core/src/display/shape/primitive/def/class.rs

```diff
@@ -52,7 +52,14 @@ impl canvas::Draw for AnyShape {
 #[derive(Debug, Derivative, Shrinkwrap)]
 #[derivative(Clone(bound = ""))]
 pub struct ShapeRef<T> {
+    #[shrinkwrap(main_field)]
     rc: Rc<T>,
+    id: ShapeId,
+}
```

Comment:
We are using this patten really often. Wouldn't it be nice to have a type `RcWithId` that Derefers to Rc?

Reply:
You've given a thumbs up here, but the code is not changed – how should I interpret it? :)